blog-posts/emails/article.md

```diff
@@ -0,0 +1,329 @@
+  - ❌ Over **0.5%** complaint rate, you will be **prevented from sending emails** until investigation.
+
+Obviously, you don't want your account to be blacklisted, but I would also advise you to act before being put under review. To prevent all this bad stuff from happening to you, here are two of my personal tips:
+
+### Set up alarms to be warned before its to late
+
+Cloudwatch alarms can be set up to monitor both the bounce and complaint rates, and send you notifications / take actions when they reach a dangerous level. These cloudwatch alarms can be set up at **account level** or **configuration set level**. Allowing you to either monitor your whole accounts reputation, or the reputation linked to specific SES identities.
+
+Let me advise you to **create a configuration set linked to the SES Identity you send emails from**, and monitor it specifically. It will allow you to identify faster where problems are coming from, and possibly shut down only parts of your messaging infrastructure if needed.
+
+_Quick example of alarm configuration that could be used to monitor bounce rate. It will trigger an alarm if bounce rate breached 4% during the last 30 minutes:_
+
+```json
+{
+  "...": "...",
+  "MetricName": "Reputation.BounceRate",
+  "Namespace": "AWS/SES",
+  "Statistic": "Average",
+  "Dimensions": [
+    {
+      "Name": "ses:configuration-set",
+      "Value": "<your_configuration_set_name>"
+    }
+  ],
+  "Period": 300,
+  "EvaluationPeriods": 6,
+  "DatapointsToAlarm": 1,
+  "Threshold": 0.04,
+  "ComparisonOperator": "GreaterThanThreshold",
+  "TreatMissingData": "notBreaching",
+  "...": "..."
+}
+```
+
+Next step is to plug a SNS topic into both reputation metrics alarms, and to either :
+
+- Subscribe your devs email address to the topic to be warned in case of issue.
+- Subscribe a lambda automatically shutting down email sending in your configuration set.
+
+![Alarms schema](./assets/alarms.png 'Alarms schema')
+
+_Disabling a configuration set using Typescript and the AWS SDK v3:_
+
+```ts
+const client = new SESClient({});
+await client.send(
+  new UpdateConfigurationSetSendingEnabledCommand({
+    Enabled: false,
+    ConfigurationSetName: '<your_configuration_set_name>',
+  }),
+);
+```
+
+### Avoid sending emails to problematic recipients
+
+With these alarms, you already have a way to quickly react to issues concerning reputation. But you can do more to avoid sending multiple emails to addresses harming your reputation.
+
+One way to achieve this is to add bouncing or complaining addresses to a **suppression list**. Three types of suppression lists exist :
+
+- Global level (AWS as a whole)
+- Account level
+- Configuration set level
+
+Like for your alarm, I advise you to be as specific as possible and to **setup a a configuration set level suppression list**.
+
+_Sample of configuration set config enabling suppression list of bounces and complaints:_
+
+```json
```

Comment:
I don't really understand what this configuration does. Does it allow you to manually put email addresses in a suppression list ?